Documentation/helm-operator.md

```diff
@@ -106,6 +106,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.enableOMAPGenerator`           | EnableOMAP generator deploys omap sidecar in CSI provisioner pod, to enable it set it to true                               | `false`                                                   |
```

Comment:
How about documentation on the feature about how to use it end-to-end?

Reply:
> How about documentation on the feature about how to use it end-to-end?

I was thinking about doing it in the pr adding cephNFS provisioner, which should be pretty soon.
It's being added to ceph-csi [here](https://github.com/ceph/ceph-csi/pull/2948)

Reply:
The nfs plugin isn't useful by itself, right? So we need to make sure this gets in the same release with the provisioner PR. How about we hold this PR and either add more commits for the provisioner, or merge them around the same time? We are targeting the first week of April for the v1.9 release, I assume that's the target. The nfs provisioner is intended to be with ceph-csi v1.6 in the same timeframe?

Reply:
I requested Rakshith send the PRs separately so they can be reviewed more easily. This is the easiest PR to work on because the components are ready today. But I do agree that this PR is not useful until the provisioner is also deployed by Rook. I would prefer to keep these PRs separate and merge this PR after the provisioner is deployed by Rook. The export provisioner is useful even without the nodeplugin.